jkube-kit/config/service/src/main/java/org/eclipse/jkube/kit/config/service/openshift/OpenshiftBuildService.java

```diff
@@ -152,6 +154,11 @@ public void build(ImageConfiguration imageConfig) throws JKubeServiceException {
         }
     }
 
+    @Override
+    public void push(Collection<ImageConfiguration> imageConfigs, int retries, RegistryConfig registryConfig, boolean skipTag) throws JKubeServiceException {
+        // Do nothing. Image is pushed as part of build phase
```

Comment:
Does it make sense to add some `log.info` here to notify the user?

I think we are doing this in the Mojo, but it's probably better to move that here (think of future gradle plugin)